tests/integration/test_charm.py

```diff
@@ -18,6 +20,88 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_admin_reachable(app: ops.model.Application, ops_test: OpsTest):
+    """
+    arrange: build and deploy the charm.
+    act: nothing.
+    assert: that the admin of the API is reachable.
+    """
+    # Application actually does have units
+    unit = app.units[0]  # type: ignore
+
+    # Mypy has difficulty with ActiveStatus
+    assert unit.workload_status == ops.model.ActiveStatus.name  # type: ignore
+
+    for _, unit_ip in enumerate(await tests.integration.helpers.get_unit_ips(ops_test, unit)):
+        url = f"http://{unit_ip}:8080/static/admin/css/base.css"
+        try:
+            response = requests.head(url, timeout=5)
+            response.raise_for_status()
+            logger.info("File found at %s", url)
+        except requests.exceptions.HTTPError as e:
+            logger.info("HTTP Error: %s", e)
```

Comment:
Will the test fail in case of HTTP error?

Reply:
You're right: it would not fail. I've removed the try/except block